docs/try_argo_cd_locally.md

```diff
@@ -51,6 +60,17 @@ To log in to the ArgoCD UI, you'll need the default admin password. You can retr
 ```bash
 kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath='{.data.password}' | base64 -d
 ```
+!!! note "PowerShell users"
+    The `base64 -d` command is not available in Windows PowerShell.
+
+    To decode the password manually, use:
+
+    ```powershell
+    [System.Text.Encoding]::UTF8.GetString(
+      [System.Convert]::FromBase64String("<BASE64_VALUE>")
```

Comment:
```suggestion
    [System.Text.Encoding]::UTF8.GetString([System.Convert]::FromBase64String((kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath='{.data.password}')))
```

a one-liner like below would be more convenient for users to avoid manual copy and paste?